op-e2e/setup.go

```diff
@@ -361,6 +355,7 @@ func (sys *System) SetBatchInboxToV2(t *testing.T) {
 	_, err = sysCfgContract.SetBatcherHash(sysCfgOwner, v2BatcherHash)
 	require.NoError(t, err)
 	log.Info("SystemConfig contract updated.")
+	time.Sleep(3 * time.Second)
```

Comment:
Why the sleep here, as opposed to the call point?

Reply:
It was to avoid forgetting to call sleep. But I am not sure this sleep function is needed because the call to the smart contract wait for the corresponding transaction to be mined. I will try without it.

Reply:
The call to `time.Sleep` was removed in 4885d666d0422d7c53d7bbcb332d20dd12f24fc4 and apparently this is not a problem.